huntsman/filterwheel/pyro.py

```diff
@@ -0,0 +1,98 @@
+from threading import Event
+
+from pocs.filterwheel import AbstractFilterWheel
+
+
+class FilterWheelEvent(Event):
+    """Interface to the move_event of a remote filterwheel.
+
+    Methods of the AbstractFilterWheel base class make use of a _move_event. In order
+    for those methods to work with a Pyro filterwheel we need to replace that Event with
+    an inteface to the event of the remote filterwheel.
+    """
+    def __init__(self, proxy):
+        self._proxy = proxy
+
+    def set(self):
+        self._proxy.filterwheel_event_set()
+
+    def clear(self):
+        self._proxy.filterwheel_event_clear()
+
+    def is_set(self):
+        return self._proxy.filterwheel_event_is_set()
+
+    def wait(self, timeout=None):
+        return self._proxy.filterwheel_event_wait(timeout)
+
+
+class FilterWheel(AbstractFilterWheel):
+    """ Class representing the client side interface to the Filterwheel of a distributed camera. """
+    def __init__(self,
+                 name='Pyro Filterwheel',
+                 model='pyro',
+                 camera=None,
+                 **kwargs):
+
+        # Need to get filter names before calling base class constructor.
+        filter_names = camera._proxy.filterwheel_filternames
+        kwargs['filter_names'] = filter_names
+        super().__init__(name=name, model=model, camera=camera, **kwargs)
+        self.connect()
+
+##################################################################################################
+# Properties
+##################################################################################################
+
+    @property
+    def is_connected(self):
+        """ Is the filterwheel available """
+        return self._proxy.filterwheel_is_connected
+
+    @property
+    def is_moving(self):
+        """ Is the filterwheel currently moving """
+        return self._proxy.filterwheel_is_moving
+
+    @property
+    def is_ready(self):
+        # A filterwheel is 'ready' if it is connected and isn't currently moving.
+        return self._proxy.filterwheel_is_ready
+
+    @AbstractFilterWheel.position.getter
```

Comment:
Should this be here still? Same for next property.

Reply:
They are needed.